content/blog/posts/student-story-whitehet.rst

```diff
@@ -0,0 +1,54 @@
+Reflections on My Time at the Open Source Lab, Travis Whitehead
+===============================================================
+:author: Travis Whitehead
+:date: 2019-04-24
+:slug: student-story-whitehet
```

Comment:
Please add the following here:
```
:tags: student-stories
```

Reply:
You still need to add this. It should show up [here](http://osuosl-pelican-251.staging.osuosl.org/tag/student-stories/) once deployed. And yes, this is separate from the ``:tag`` line.